### DISCUSSION TITLE
Domain is improperly configured (but works)

### DISCUSSION THREAD

**Discussion by JonasVautherin** [community/community#29787] [Other Feature Feedback, Questions, & Ideas]
Using my custom domain with Pages, I get the following error in the Settings:

```
<domain> is improperly configured
Domain does not resolve to the GitHub Pages server. For more information, see [documentation](https://docs.github.com/articles/setting-up-a-custom-domain-with-github-pages/) (NotServedByPagesError).
```

However, it does work the way I want it: "www.mydomain.com" points to the Pages, and "mydomain.com" points to my VPS. Is it illegal to point a subdomain to a different IP?

**Comment by airtower-luna** [comment]
That should work, if you have set the full `www.mydomain.com` as your custom domain: https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site#configuring-a-subdomain If you have only set `mydomain.com` the error makes sense.

**Reply by JonasVautherin** [reply]
I have set `www.mydomain.com` as the custom domain, and confirmed the CNAME with:

```
% dig www.mydomain.com +nostats +nocomments +nocmd                                                                                        [0]
;www.mydomain.com.			IN	A
www.mydomain.com.		2278	IN	CNAME	mydomain.github.io
```

**Reply by airtower-luna** [reply]
The CNAME should point at `YOUR-USERNAME.github.io` (so `jonasvautherin.github.io` in your case), the target has nothing to do with the domain you're configuring.

**Reply by JonasVautherin** [reply]
Sure, sorry. It's actually a GitHub organization, so `mydomain` is more `myorg` (and not my username). That's why  I have `www.myorg.com` pointing to `myorg.github.io`.

Is the fact that it is an organization a problem?

**Reply by airtower-luna** [reply]
I don't see why it would, the documentation describes both exactly the same way. :thinking: 

Is the DNS change recent? It might take a while to work its way through caches.

**Reply by JonasVautherin** [reply]
That was a month ago :confused:. What's weird is really that it all works the way I want it. I just have this warning, which I would like to resolve :innocent:.

**Reply by airtower-luna** [reply]
Okay, that should definitely be long enough. :sweat_smile: In that case I'd [send a bug report to GitHub support](https://support.github.com/contact/bug-report), in the hope they can figure out what's going on.

**Comment by jfullstackdev** [comment]
I think it's not your domain because it's a site to register domain and a website builder etc.,

if you will be doing a redirect, then it might violate the rules of GitHub, like starting to redirect to a site that can be considered as business

but not very sure of that, that's just my opinion
